django_netjsonconfig/apps.py

```diff
@@ -42,6 +42,24 @@ def check_settings(self):
                                        'NETJSONCONFIG_REGISTRATION_ENABLED to False')
 
     def ready(self):
+        from django.contrib import admin
+        from django.contrib.admin import sites
+        from .views import schema
+        from django.conf.urls import url
+
+        class CustomAdminSite(admin.AdminSite):
+
+            def get_urls(self):
+                urls = super(CustomAdminSite, self).get_urls()
+                custom_urls = [
+                    url(r'^netjsonconfig/schema\.json$', schema, name='schema'),
```

Comment:
can we do this in the main base admin file to keep consistency with what we already have?

https://github.com/openwisp/django-netjsonconfig/blob/master/django_netjsonconfig/base/admin.py#L81

In theory if you do this there you won't need to do the same thing in openwisp-controller, right?

Reply:
Thanks, I didn't see the existing `get_urls` function.
Moved the netjsonconfig there! :smile: